src/Nox.Types/Types/Temperature/Temperature.cs

```diff
@@ -90,34 +63,38 @@ internal override ValidationResult Validate()
     }
 
     /// <summary>
-    /// Converts the this instance to its equivalent string representation using InvariantCulture format.
+    /// Converts <see cref="Temperature.TemperatureValue"/> to value in <see cref="TemperatureUnit.Celsius"/>.
```

Comment:
You removed the property TemperatureValue and didn't update the comment.

Reply:
done